Documentation/Configuration/Reference/TxSolrIndex.rst

```diff
@@ -380,6 +380,25 @@ Example:
 
     plugin.tx_solr.index.queue.tt_news.attachments.fields = news_files
 
+queue.[indexConfig].recursiveUpdateFields
+-----------------------------------------
+
+:Type: String
+:TS Path: plugin.tx_solr.index.queue.[indexConfig].recursiveUpdateFields
+:Since: 6.1
+:Default: Empty
+
+Allows to define a list of additional fields from the pages table that will trigger an recursive update of
```

Comment:
from the **table to index** ... since it's not specific to just pages, or is it?